test/test_transforms.py

```diff
@@ -272,9 +264,41 @@ def test_librosa_consistency(self):
 #                                             n_mels=n_mels)
 
         librosa_mfcc = scipy.fftpack.dct(db_librosa, axis=0, type=2, norm='ortho')[:n_mfcc]
-        torch_mfcc = mfcc_transform(sound).squeeze().cpu().numpy().T
-
-        self.assertTrue(np.allclose(torch_mfcc, librosa_mfcc, atol=5e-3))
+        torch_mfcc = mfcc_transform(sound).squeeze().cpu().t()
+
+        self.assertTrue(torch.allclose(torch_mfcc.type(torch.double), torch.from_numpy(librosa_mfcc), atol=5e-3))
+
+    def test_(self):
+        kwargs1 = {
```

Comment:
What is the reasoning behind these particular parameters?

Reply:
Instead of calling into "self._test_..." make "self._test_..." a local function "_test_" of def "test_" and rename "test_" to something more meaningful. This should show up as a separate test on the command line when run with "python test_transforms.py --verbose". The local function is an abstraction to cover multiple arguments as you do here.

Reply:
Done. Not sure about those parameters, I just increased/decreased the sample rate, number of fourier/mel bins arbitrarily.

Reply:
Alright, let's revisit that as a follow-up PR.